### ISSUE TITLE
Slothbot bug

### ISSUE THREAD

**Issue by GribixTugaZ** [NullSystemWorks/mtadayz#60]
When trying to get into my server and downloading everything this error appears:

http://i.imgur.com/0ecg5Sa.png

**Comment by ghost** [comment]
Are you using an external HTTP server for delivering the resource files?

**Comment by GribixTugaZ** [comment]
As far as I know no. I'm only uploading the files to the ftp. That's all. I may contact the hosting company and ask about it.

**Comment by mtadayz** [comment]
That can happen for two reasons:

- The file is missing
- the httpserver cannot be accessed by the server

If it's the latter, check your serverconfig and look for httpserver and httpdownloadurl, maybe something's not right there. If that fails, I'd just delete the slothbot resource (MTA San Andreas 1.5/mods/deathmatch/resources or MTA San Andreas 1.4/mods/deathmatch/resources) and try it again.

Basically, this is not a bug caused by DayZ. Closing this issue.
